src/components/dataset/overview/linked-forms.vue

```diff
@@ -0,0 +1,79 @@
+<!--
+Copyright 2023 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+
+<template>
+  <summary-item id="linked-forms" icon="link">
+    <template #heading>
+      {{ linkedForms.length }}
+    </template>
+    <template #body>
+      <p>{{ $tc('formsConsumeData', linkedForms.length) }}</p>
+      <table v-if="linkedForms.length > 0" class="table">
+        <tbody>
+          <tr v-for="(form) in linkedForms" :key="form.xmlFormId">
+            <td>
+              <router-link :to="publishedFormPath(projectId, form.xmlFormId)" :title="form.name">{{ form.name }}</router-link>
```

Comment:
Could you add assertions about this link in testing, of its `to` prop and its `title` attribute? And add another test that the `to` prop is correct if the user is a project viewer?